src/test/java/net/datafaker/providers/base/BaseFakerTest.java

```diff
@@ -54,7 +56,7 @@ protected List<String> getBaseList(String key) {
 
     @ParameterizedTest(name = "{0}")
     @MethodSource("providerListTest")
-    protected void testProviderList(TestSpec testSpec, TestInfo testInfo) {
```

Comment:
not sure about that change...
when it is protected it allows to use this in other projects to test custom fakers...

Reply:
Okay, will restore